0002-DataCredits.md

```diff
@@ -0,0 +1,160 @@
+- Start Date: 2020-02-26
+- HIP PR: <!-- leave this empty -->
+- Tracking Issue: <!-- leave this empty -->
+
+# Summary
+[summary]: #summary
+
+Data Credits have a fixed value of $.00001 per 24 bytes. There were original 
+consideration to charge for Time-on-Air, but a fixed price for data usage was
+settled upon to simplify the experience of network users.
+
+However, how are bytes counted? Between the LoRa Physical Header, the 
+LoRaWAN/MAC header, and the MAC Payload, FOpts and FRMPayload there are several
+ways of deciding how to meter bytes.
+
+This proposal here is to charge for **FOpts** and **FRMPayload** exclusively,
+but to charge 1 DC per Join uplink and 1 DC per Join-Accept downlink.
+
+FOpts are fifteen optional bytes used almost exclusiveely for MAC Commands.
+
+FRMPayload are essentially the application data.
+
+By defining things this way, we can keep the data calculations very simple for
+network users, while also protecting against simple ways to move additional
+data "for free".
+
+In addition, we propose to round up DC costs. Therefore, if I transmit a DataUp
+frame with 55 bytes: `55/24.0 ~= 2.3 DCs => 3 DCs` burned on behalf of the
+Hotspot delivering the packet.
+
+# Motivation
+[motivation]: #motivation
+
+Striking the balance here means being fair to both sides of the market: Users and
+Operators.
+
+# Stakeholders
+[stakeholders]: #stakeholders
+
+* Helium Network Users
+* Helium Network Operators
+
+# Detailed Explanation
+[detailed-explanation]: #detailed-explanation
+
+When an Operator transmits or receives a packet, they are unable to transmit 
+and/or receive packets. As such, we must fight against exploits and charge a fee
+for Join and Join-Accept frames.
+
+When a User considers the Helium Network, they know about their application, and
+so have ideas of how large their payloads are, but may not know of the nuances
+of LoRa or LoRaWAN versus other modulations or protocols. To help them make the
+business decision to use the Helium Network, we believe it to simply the data cost
```

Comment:
the grammar in this sentence is a little odd

Reply:
Agreed. I did some rephrasing